contracts/BondStorage.sol

```diff
@@ -261,7 +273,7 @@ contract BondStorage is AccessControl {
     // subtracted with the appropriate amount and the claim counter is increased with the
     // sum of the principals and the their respective accrued interest, all in TST.
     // If the user has no bonds active, the isActive will be switched to false.
-    function refreshBondStatus(address _user) external {
+    function refreshBondStatus(address _user) external onlyWhitelisted {
```

Comment:
this needs to be vanilla public, no need to protect a user from refreshing anyone's bond status

Reply:
that seems fair. i just thought as we had a route to this in the operator it made sense to force the use of that route, but i can't see any harm in this being called directly